src/components/WindowThumbnailSettings.js

```diff
@@ -34,42 +34,65 @@ export class WindowThumbnailSettings extends Component {
    * @return {type}  description
    */
   render() {
-    const { thumbnailNavigationPosition, t } = this.props;
+    const {
+      classes, handleClose, t, thumbnailNavigationPosition,
+    } = this.props;
 
     return (
       <>
-        <Typography>{t('thumbnails')}</Typography>
-        <RadioGroup aria-label={t('position')} name="position" value={thumbnailNavigationPosition} onChange={this.handleChange} row>
+        <ListSubheader
+          role="presentation"
+          selected={false}
+          disabled
+          tabIndex="-1"
+        >
+          <Typography>{t('thumbnails')}</Typography>
```

Comment:
These `Typography` elements cause the headers to be quite small (as noticeable in the gif).  Not sure if there is a `variant` I should be using here or what.